tests/infra/remote.py

```diff
@@ -651,7 +644,7 @@ def __init__(
             f"--snapshot-dir={self.snapshot_dir_name}",
             f"--node-cert-file={self.pem}",
             f"--host-log-level={host_log_level}",
-            election_timeout_arg,
+            f"--raft-election-timeout-ms={raft_election_timeout_ms}",
```

Comment:
I think we should take the config migration opportunity to rename this to "--cft-*".

Reply:
Yep, it's on the list!